modular_skyrat/modules/importedvendors/code/vendor_snacks.dm

```diff
@@ -0,0 +1,275 @@
+	AddElement(/datum/element/food_trash, trash_type, FOOD_TRASH_POPABLE)
+
+/obj/item/trash/vendor_trash/ricecracka
+	name = "empty rice crackers bag"
+	desc = "You never did find out what flavor that was supposed to be, did you?"
+	icon_state = "ricecracka_trash"
+
+/obj/item/food/vendor_snacks/mochi_icecream
+	name = "mochi icecream balls - vanilla"
+	desc = "A six pack of mochi icecream, which is to say vanilla icecream surrounded by mochi. Comes with small plastic skewer for consumption."
+	icon_state = "mochiice"
+	trash_type = /obj/item/trash/vendor_trash/mochi_icecream
+	food_reagents = list(/datum/reagent/consumable/nutriment = 3, /datum/reagent/consumable/ice = 3)
+	tastes = list("rice cake" = 2, "vanilla" = 2)
+	foodtypes = JUNKFOOD | DAIRY | GRAIN
+	custom_price = PAYCHECK_LOWER * 1.2
+
+/obj/item/food/vendor_snacks/mochi_icecream/matcha
+	name = "mochi icecream balls - matcha"
+	desc = "A six pack of mochi icecream, which is to say matcha icecream surrounded by mochi. Comes with small plastic skewer for consumption."
+	icon_state = "mochiice_green"
+	food_reagents = list(/datum/reagent/consumable/nutriment = 3, /datum/reagent/consumable/ice = 1, /datum/reagent/consumable/tea = 2)
+	tastes = list("rice cake" = 1, "bitter matcha" = 2)
+	custom_price = PAYCHECK_LOWER * 1.5
+
+/obj/item/food/vendor_snacks/mochi_icecream/matcha/examine_more(mob/user)
+	. = ..()
+	. += span_notice("A small label on the container specifies that the icecream is made using only culinary grade matcha grown outside of the sol system.")
+	return .
+
+/obj/item/trash/vendor_trash/mochi_icecream
+	name = "empty mochi icecream tray"
+	desc = "Somehow, that tiny plastic skewer it came with has gone missing."
+	icon_state = "mochiice_trash"
+
+/obj/item/reagent_containers/cup/glass/waterbottle/tea
+	name = "bottle of tea"
+	desc = "A bottle of tea brought to you in a convenient plastic bottle."
+	icon = 'modular_skyrat/modules/importedvendors/icons/imported_quick_foods.dmi'
+	icon_state = "teabottle"
+	inhand_icon_state = "bottle"
+	list_reagents = list(/datum/reagent/consumable/tea = 40)
+	cap_icon_state = "bottle_cap_tea"
+	flip_chance = 5 //I fucking dare you
+	custom_price = PAYCHECK_LOWER * 1.5
+	fill_icon_state = null
+
+/obj/item/reagent_containers/cup/glass/waterbottle/tea/astra
+	name = "bottle of tea astra"
+	desc = "A bottle of tea astra, known for the rather unusual tastes the leaf is known to give when brewed."
+	icon_state = "teabottle_blue"
+	list_reagents = list(
+		/datum/reagent/consumable/tea = 25,
+		/datum/reagent/medicine/salglu_solution = 10, // I know this looks strange but this is what tea astra grinds into, tea in the year 25whatever baby
+		/datum/reagent/consumable/nutriment/vitamin = 5,
+	)
+	custom_price = PAYCHECK_LOWER * 3
+
+/obj/item/reagent_containers/cup/glass/waterbottle/tea/strawberry
+	name = "bottle of strawberry tea"
+	desc = "A bottle of strawberry flavored tea, does not contain any actual strawberries."
```

Comment:
```suggestion
	desc = "A bottle of strawberry flavored tea; does not contain any actual strawberries."
```
This one might have been harder to catch with my earlier explanation so I'll just do it here; the first half explains the item, the second adds on a description of the first item but isn't meant to be immediately paired. Think of it as giving a moment to process before you drop the punchline